### DISCUSSION TITLE
Playbook mysql errors

### DISCUSSION THREAD

**Discussion by blarson1105** [Security-Onion-Solutions/securityonion#1592] [Unsupported Versions] [answered]
I was able to successfully install and run SO 2.3.  However, I have a couple colleagues trying to install it on a vm and they are getting mysql errors.  When comparing the sosetup.log files, we see the following:

[INFO    ] Executing state module.run for [create_playbookdbuser]
[ERROR   ] MySQL Error 2003: Can't connect to MySQL server on '192.168.197.159' (113)
[ERROR   ] MySQL Error: Unable to fetch current server version. Last error was: "MySQL Error 2003: Can't connect to MySQL server on '192.168.197.159' (113)"
[INFO    ] No changes made for ['mysql.user_create']

I have tried to isolate the issue by executing previous solutions for the RC's.

salt-call state.apply playbook.db_init
so-playbook-restart
so-playbook-ruleupdate

No luck.  Any help would be appreciated.

**Comment by defensivedepth** [comment, marked as answer]
There have been some changes with GA - please try the following:

`sudo salt-call state.apply playbook.db_init`
`sudo salt-call state.apply playbook.automation_user_create`

Do you get any errors at this point?

**Comment by dannatofrenk** [comment]
I solved doing an iptables -F while setp still looping on the attempts to connect to MySQL. It worked. I am installing on a bare Centos 7 minimal. Don't know what will happen to mysql connections when i restart the firewall. It's taking forever to finish setup. I think the connection to mysql  should try to localhost by default, instead of trying the lan ip. 
Regards 
F

**Reply by defensivedepth** [reply]
@dannatofrenk This is an old issue, and a number of updates have been released since then. If you are having issues, please start a new Discussion and describe what you are seeing.

Thanks!
